### PR TITLE
first pull req

### PR DESCRIPTION
**Pull request by MohanKona29**
final commit

### CODE REVIEW

**Comment by MohanKona29** [comment]
Ma'am , u can review now!

File: README.md [hunk 1]
```diff
@@ -22,3 +22,10 @@ Things you may want to cover:
 * Deployment instructions
 
 * ...
+
```
Comment (susanjacob) [inline README.md:25]: Please mention the rails and ruby version in README.

File: app/models/post.rb [hunk 2]
```diff
@@ -1,3 +1,5 @@
 class Post < ApplicationRecord
-  has_many :comments, dependent: :destroy
-end
+  belongs_to :user
+  has_many :comments
+  validates :title, :body , presence: true
```
Comment (susanjacob) [inline app/models/post.rb:4]: user should also be validated for presence.

File: app/models/comment.rb [hunk 3]
```diff
@@ -1,3 +1,4 @@
 class Comment < ApplicationRecord
+  validates_presence_of :post_id
```
Comment (susanjacob) [inline app/models/comment.rb:2]: Shouldn't there be an association with user?

**Review by susanjacob — CHANGES_REQUESTED** [review]
Hi @MohanKona29 ,
Please amend the first 2 commits with proper messages.
Create a migration to add database level validations for comments, posts, and users, for all not null cases.

**Pull request merged**